tool/vfx/vfxParser.cpp

```diff
@@ -94,6 +101,39 @@ bool isArrayAccess(const char *str);
 // in the string
 char *getWordFromString(char *str, char *wordBuffer);
 
+// =====================================================================================================================
+// Splits the input string by modifying it in place. Returns a vector of (inner) fragment strings. This is a thread-safe
```

Comment:
Thanks, I wrote `split` so that it did what the code expected it to and missed that `strtok` takes a set of delimiters instead of a single one. I changed the code so that treats it as a set of delimiter characters, but I did not pursue 100% compatibility with `strtok` (there are some corner cases around leading/trailing delimiters, empty delimiter sets, etc., that do not matter for Vfx).

I added unit tests like you suggested.